### DISCUSSION TITLE
how do you install custom ssl cert

### DISCUSSION THREAD

**Discussion by anmiefan** [AzuraCast/AzuraCast#5768] [Q&A]
i need help to install custom ssl cert i am running the docker ver of AzuraCast and Rolling ver but the all the info about it i found is from back in 2020 and i don't know if the info about ssl from AzuraCast Docs is up to date please and also how do you exe the command to set the custom ssl cert and i need help fast

**Comment by BusterNeece** [comment]
@anmiefan See this section on our docs: https://docs.azuracast.com/en/administration/ssl-and-lets-encrypt#using-a-custom-certificate

The updated information for releases version 0.16.0 or greater is still valid with the newest version.

**Reply by anmiefan** [reply]
i just saw that now  but how you add it to the web container it don't show the command to add or update to conainer i tried some other things but it didn't work like i added it to docker-compose.yml that didn't work and its not picking up the changes from the docker-compose.override.yml file so i don't knoe if this works i downed the contianer and started back up and still nothing i have another is to install openssh in the container and get in to the file system and copy the ssl files in the ssl folder 

update i got working i didn't see the updated doc untile now i didn't refrash my browser its working now i also digged insde the contianer its self so everything works thanks for updateing the doc yesterday
